aviary/subsystems/geometry/gasp_based/fuselage.py

```diff
@@ -182,7 +162,7 @@ def compute(self, inputs, outputs):
         LoverD_nose = inputs[Aircraft.Fuselage.NOSE_FINENESS]
         LoverD_tail = inputs[Aircraft.Fuselage.TAIL_FINENESS]
         cockpit_len = inputs[Aircraft.Fuselage.PILOT_COMPARTMENT_LENGTH]
-        fus_SA_factor = inputs[Aircraft.Fuselage.WETTED_AREA_FACTOR]
+        fus_SA_factor = inputs[Aircraft.Fuselage.WETTED_AREA_SCALER]
```

Comment:
Should change this variable name to `fus_SA_scaler`

Reply:
Done.